cypress/fixtures/examples/mock.json

```diff
@@ -62,6 +62,32 @@
       }
     }
   },
+  "group": {
+    "Description": "The group assignment of the participant in a study.",
+    "Levels": {
+      "ADHD": {
+        "Description": "Attention deficit hyperactivity disorder",
+        "TermURL": "snomed:406506008"
+      },
+      "PD": {
+        "Description": "Parkinsons",
+        "TermURL": "snomed:870288002"
+      }
```

Comment:
Are these levels wrong / mismatched from the `mock.tsv` values for `"group"` on purpose? i.e. I would expect to see `"HC"` and `"Patient"` show up here.

Reply:
Yes, let me fix that before we merge